version_info.json

```diff
@@ -1,8 +1,8 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220726.216",
-    "iree-runtime": "20220726.216",
-    "iree-tools-xla": "20220726.216",
+    "iree-compiler": "20220809.230",
+    "iree-runtime": "20220809.230",
+    "iree-tools-xla": "20220809.230",
     "jax": "0.3.15",
```

Comment:
Can we bump this to latest as well? Looks like 0.3.16 is on PyPi now, but jaxlib still only has 0.3.15. What's up with that?

![image](https://user-images.githubusercontent.com/5732088/184733611-bb0de1e8-107e-47b7-974b-e9cfcf1a878e.png)
![image](https://user-images.githubusercontent.com/5732088/184733647-763254ea-b02e-4be6-a57c-70e7bc976ed2.png)

Reply:
I was hoping to test 0.3.16 with the auto-update tool.